src/VisualStudio/Core/Impl/Options/GridOptionPreviewControl.xaml.cs

```diff
@@ -349,9 +355,13 @@ private static void DotNetCodeStyleOptions_GenerateEditorconfig(OptionSet option
                 {
                     editorconfig.AppendLine("omit_if_default:" + curSetting.Notification.ToString().ToLower());
                 }
+                else if (curSetting.Value == AccessibilityModifiersRequired.Always || curSetting.Value == AccessibilityModifiersRequired.Never)
+                {
+                    editorconfig.AppendLine(curSetting.Value.ToString().ToLower() + ":" + curSetting.Notification.ToString().ToLower());
```

Comment:
i just realized something awful.  You'll need to do .ToString and .ToLower passing the ordinal culture.  Otherwise, ToLower can do the wrong thing if someone is in a culture different than english.  :(

team members can shed some more light on this...

Reply:
In the (bug thread)[https://github.com/dotnet/roslyn/issues/28584] I opened, a user expressed their worries for localization in .editorconfig. What are your thoughts on this?

Reply:
Note: this does not affect me personally and probably isn't a big issue in this case but it's definitely something to think about wrt to localization, especially with generated code.

I think I've seen @MaStr11 hit a similar issue when contributing to this repo where a generated file was modified to use his own language and there was no way to configure that - in the end the file had to be regenerated on another user's machine as a workaround.

Reply:
It was this commit that corrected the changes caused by my German VS when I saved a resx file:
https://github.com/dotnet/roslyn/pull/23587/commits/300c92a8cc1c33ef61c8bd815080af9f5a04eefe

The resx tool does localized XML doc comments.

Reply:
This is a different matter.  This specific comment from me is that .ToLower is literally incorrect here.  It means that you might not end up with the exact values of ```true/false/whatever```.  You will be dependent on the locale you are in, which means maybe some other character is substituted in.  We would then be unable to parse this on the other side.  This is a different issue vs whether or not the *headers* are localized are not.

So, two issues:

1. should we localize the headers?  IMO, yes. We have long precedence of emitting things that way, and i don't think we should be changing our policy here without strong evidence that we should.
2. should we localize the values?  Absolutely not.  We should not use anything culture sensitive with the actual values.  These should be written and read in exactly one supported format.

Reply:
I understand the specific comment here and my comment doesn't apply to that. I just noticed that @allisonchou brought up my comment in the issue here